### PR TITLE
doc(news blog): adds news blog for the release of T-X24.05

### PR DESCRIPTION
**Pull request by jSchuetz88**
Pull-Request for issue: #903

- PR adds the respective blog-post for R24.05

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: static/img/Tractus-X_Release24.05.png
Comment (HFocken) [inline static/img/Tractus-X_Release24.05.png:1]: Now that I see it, I would say change it to:

"Our most significant release yet"

**Review by HFocken — CHANGES_REQUESTED** [review]

**Comment by maximilianong** [comment]
Please do not forget to add this news to the slider on the home/main page.

**Review by maximilianong — APPROVED** [review]

File: static/img/Tractus-X_Release24.05.png
Comment (jSchuetz88) [reply static/img/Tractus-X_Release24.05.png:1]: Done

**Comment by jSchuetz88** [comment]
> Please do not forget to add this news to the slider on the home/main page.

done

File: blog/2024-05-28-release-24-05.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - name: Hanno Focken
+    title: Ecosystem & Kits
+    url: https://github.com/hfocken
+    image_url: https://github.com/hfocken.png
+    email: hanno.focken@catena-x.net
+---
+
+![Release Image](@site/static/img/Tractus-X_Release24.05.png)
+
+We are excited to announce the release of Tractus-X 24.05! This release builds the foundation for one of the most significant updates to our Catena-X ecosystem to date. Highlights include an update to the Multi-Opco solutions, the full implementation of the Industry Core KIT as well as the advancement of Self Sovereign Identity concepts, e.g. Multi-Wallet solutions. Conjointly, it makes several improvements to the user-friendliness of KITs based on feedback received to date, making Tractus-X overall more accessible and welcoming to newcomers and veterans alike. A big thank you to all contributors and committers for bringing this milestone to life and keep Tractus-X as one of the most significant OS projects in the automotive industry.
```
Comment (danielmiehle) [inline blog/2024-05-28-release-24-05.md:17]: I would suggest to replace: ecosystem = data space

Comment (danielmiehle) [inline blog/2024-05-28-release-24-05.md:17]: AFAIK: Multi-Wallet Solutions is not enabled, yet. Please verify with @jjeroch .

Comment (danielmiehle) [inline blog/2024-05-28-release-24-05.md:17]: I would suggest replace: OS = open-source.
Could be confused with operating system.

**Review by danielmiehle — CHANGES_REQUESTED** [review]

File: blog/2024-05-28-release-24-05.md (hunk 1 above)
Comment (jSchuetz88) [reply blog/2024-05-28-release-24-05.md]: Done

Comment (jSchuetz88) [reply blog/2024-05-28-release-24-05.md]: Done

Comment (jSchuetz88) [reply blog/2024-05-28-release-24-05.md]: Done

**Pull request merged**